tests/chainer_tests/functions_tests/math_tests/test_sign.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+
+
+class UnaryFunctionsTestBase(unittest.TestCase):
```

Comment:
How about merging UnaryFunctionsTestBase and TestSign

Reply:
Merged the classes.